### DISCUSSION TITLE
Korge won't download

### DISCUSSION THREAD

**Discussion by ELITE08** [korlibs/korge#2306] [Q&A]
When running the command in CMD an error appears.
In Windows 11

ERROR downloading https://github.com/korlibs/universal-jre/releases/download/0.0.1/microsoft-jre-21.0.3-windows-x64.tar.xz, SHA256=5A242A630E7D26868D473BB5D3FCC4488462BFD957E2FE0061B1274BEBBAD5AB, but expected SHA256=6D16528A2201DCBE0ADDB0622F5CBE0CD6FA84AE937D3830FC1F74B32132C37B
Extracting C:\Users\79621\.korge\jre-21.tar.xz...

**Comment by soywiz** [comment]
Does this still reproduces?
